common/src/main/java/com/wynntils/models/wynnitem/parsing/WynnItemParser.java

```diff
@@ -170,17 +174,46 @@ public static WynnItemParseResult parseItemStack(
                 continue;
             }
 
-            // Look for level requirements
+            // Requirements
+            // Combat level
             Matcher levelMatcher = normalizedCoded.getMatcher(MIN_LEVEL_PATTERN);
             if (levelMatcher.matches()) {
-                level = Integer.parseInt(levelMatcher.group(1));
+                level = Integer.parseInt(levelMatcher.group("level"));
                 continue;
             }
 
+            // Class
+            Matcher classMatcher = normalizedCoded.getMatcher(CLASS_REQ_PATTERN);
+            if (classMatcher.matches()) {
+                String mark = classMatcher.group(1);
+                if (mark.contains("✖")) {
+                    allRequirementsMet = false;
+                }
+            }
+
+            // Skills
+            Matcher skillMatcher = normalizedCoded.getMatcher(SKILL_REQ_PATTERN);
+            if (skillMatcher.matches()) {
+                String mark = skillMatcher.group(1);
+                if (mark.contains("✖")) {
+                    allRequirementsMet = false;
+                }
+            }
+
+            // Misc requirements
+            Matcher miscMatcher = normalizedCoded.getMatcher(MISC_REQ_PATTERN);
+            if (miscMatcher.matches()) {
+                String mark = miscMatcher.group(1);
+                if (mark.contains("✖")) {
+                    allRequirementsMet = false;
+                }
+            }
+
             Matcher setBonusMatcher = normalizedCoded.getMatcher(SET_BONUS_PATTEN);
             if (setBonusMatcher.matches()) {
                 // Any stat lines that follow from now on belongs to the Set Bonus
                 // Maybe these could be collected separately, but for now, ignore them
+                // TODO: we can actually do this now
```

Comment:
So, basically, I want you to do this, instead of what you do now. See https://github.com/Wynntils/Artemis/pull/2343/files#r1553784277